packages/@okta/i18n/src/properties/login.properties

```diff
@@ -950,3 +951,6 @@ oie.consent.enduser.deny.label = No, it's not me
 oie.consent.enduser.title = Did you just try to sign in?
 oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
 oie.consent.enduser.deny.description = You may close this window at any time.
+
+# Browser thrown errors
+browser.error.NotAllowedError = The operation either timed out or was not allowed.
```

Comment:
append `oie.` to key

Reply:
Where is this key being used ?

Reply:
That is for the browser error that throw from the chrome/firefox or others